website/docs/collaborative_whiteboard.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 2
+slug: /collaborative_whiteboard
```

Comment:
nit: keep the same slug so that old link can work

Reply:
Made sure the slugs remain the same.